### ISSUE TITLE
Precompilation of package fails on julia 1.9

### ISSUE THREAD

**Issue by KristofferC** [coudertlab/CrystalNets.jl#13]
```
ERROR: LoadError: AssertionError: !(ismissing(fbody))
Stacktrace:
 [1] precompile_kwarg(tt::Any, kwargtypes::Any)
   @ CrystalNets ~/.julia/packages/CrystalNets/2wE3X/src/precompile.jl:61
 [2] _precompile_()
   @ CrystalNets ~/.julia/packages/CrystalNets/2wE3X/src/precompile.jl:428
 [3] top-level scope
   @ ~/.julia/packages/CrystalNets/2wE3X/src/precompile.jl:821
 [4] include(mod::Module, _path::String)
   @ Base ./Base.jl:456
 [5] include(x::String)
   @ CrystalNets ~/.julia/packages/CrystalNets/2wE3X/src/CrystalNets.jl:21
 [6] top-level scope
   @ ~/.julia/packages/CrystalNets/2wE3X/src/CrystalNets.jl:97
```

Erroring like this when a precompile statement doesn't work is a bit harsh.

**Comment by Liozou** [comment]
Reworking those precompilation statements with SnoopPrecompile has been on my TODO list for a while, I'll try to get to it sometime soon hopefully. I'll remove the overly strict `@assert` while I'm at it.
Sorry for the noise on PkgEval (I assume)!

**Comment by Liozou** [comment]
The issue causing this particular error should be fixed now so CrystalNets should work on v1.9, but I'll leave this issue open until I've switched to a more robust precompilation strategy.

**Comment by Liozou** [comment]
Closed by https://github.com/coudertlab/CrystalNets.jl/pull/14
